src/query/block/types.go

```diff
@@ -96,8 +105,8 @@ type SeriesIter interface {
 type UnconsolidatedSeriesIter interface {
 	Iterator
 	SeriesMetaIter
-	// Current returns the current series for the block
-	Current() (UnconsolidatedSeries, error)
+	// CurrentUnconsolidated returns the current series for the block
+	CurrentUnconsolidated() (UnconsolidatedSeries, error)
```

Comment:
To have a single block that would satisfy both Consolidated and Unconsolidated block interfaces, otherwise it would have conflicts. Originally I had a single SeriesIterator and StepIterator that satisfied both interfaces, so had to make the change here, too, but since split it up, so can revert this